script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_8.txt

```diff
@@ -73,76 +73,76 @@ The vampire's fighting spirit has not faded.
 Descendant carrying new Principle.
 What doth blood mean to thee?"
 <Page25>
-It was a stern voice, far removed from the current situation.
+It was a stern voice, jarringly out of place in the current situation.
 Arcueid's question could be taken as a reprimand.
-After opening his eyes wide in surprise, Vlov says the following.
+After opening his eyes wide in surprise, Vlov replies.
 <Page26>
 "―――Allow me to respond with respect.
 For me, blood is the very essence of warmth.
-The heat of life. The elixir to cure my frostbite. That which turns my world and my Principle.
+The heat of life. The elixir to cure my frostbite. Around it revolves my world, and my Principle.//Blood's what makes his world go round, phrasing I chose here is a bit less literal but I think translates better to both subjects. 我が世界、我が原理を回すもの。
 Indeed. Without blood, I would freeze to death in this blistering cold."
-The man reveals his own true colors in a voice that could be described as respectful.
+The man reveals his true colors in a voice that could be described as respectful.
 <Page27>
 "...I remember now.
-A freak who hated and feared humans, drifting across an immense sea and setting up a castle to avoid them.
+A freak who hated and feared humans, who drifted across an immense sea and built a castle to avoid them.//人間を嫌い、恐れ、避けようと絶海を漂流し、城を構えた変わりもの。
 One of the Twenty-Seven Ancestors, Zaria Offenbaum.
-I heard that she ruled a land while possessing a Principle for over a thousand years without ever having children, but...... I see.
+I heard that she ruled a land while possessing a Principle for over a thousand years without ever having 'children', but...... I see.
 In other words, you killed her and took it, didn't you?"//ruby dots <殺|・><し|・><て|・><奪|・><っ|・><た|・><の|・><ね|・>
 <Page28>
-"...That's right. I am not a successor, I am simply a knight. At least I once was. Eternity and other such concepts are well beyond my comprehension."
+"...That's right. I am not a successor, I am simply a knight. Or rather, I was. Eternity and other such concepts are well beyond my comprehension."
 <Page29>
-The vampire's right hand is pointed toward his own shadow.
-From the long―――unnaturally long shadow, a huge weapon emerges.//ruby dots <得|・><物|・>
+The vampire turns his right hand toward his own shadow.
+From the long―― unnaturally long―― shadow that stretches before him emerges the hulking form of an outlandish weapon.//得物 seems a bit more ~special~ than just weapon. Ruby dots <得|・><物|・>
 <Page30>
-"But even for a family head, she was worse than unworthy.
-Therefore I took it. As a knight, I defeated the king and usurped his throne. And now this Principle has become my own personal curse."
+"However, she was utterly undeserving of the position of family head.//だが、ご当主にとっても、ソレはより不相応だった。
+Therefore I took it. As a knight, I defeated the Queen and usurped her throne. And now this Principle is mine alone."//Referring to Zaria, so queen/her?
```

Comment:
So this might be a lore point, but the flames and other chounouryoku seem to be referred to as 'curses', and given the もの ruby my impression here is that he has stolen possession of the 'curse'/power as an object/ability that belongs only to him, rather than the power being a curse upon him (a negative) which is how I would read the old version of this sentence. Or did you just mean that we should bring back the specific word curse in reference to the object?

Reply:
specific word curse. That ruby is an example of ruby that actually has meaning.